pages/index.js

```diff
@@ -1,23 +1,40 @@
+import BookmarkButton from "@/components/BookmarkButton";
 import PlantCard from "@/components/PlantCard";
+import PlantForm from "@/components/PlantForm";
 import styled from "styled-components";
 
-export default function HomePage({ plants, onToggleOwned, ownedPlantsIds }) {
+export default function HomePage({
+  plants,
+  onToggleOwned,
+  ownedPlantsIds,
+  onAddPlants,
+  displayForm,
+  setDisplayForm,
+}) {
   return (
     <div>
       <h1>{"iPlan{t}"}</h1>
+      {displayForm ? (
+        <PlantForm onAddPlants={onAddPlants} setDisplayForm={setDisplayForm} />
+      ) : (
+        ""
+      )}
       <PlantList>
-        {" "}
         {plants.map((plant) => (
           <li key={plant.id}>
-            {" "}
             <PlantCard
               id={plant.id}
               name={plant.name}
               image={plant.imageUrl}
               botanicalName={plant.botanicalName}
               onToggleOwned={onToggleOwned}
               ownedPlantsIds={ownedPlantsIds}
-            ></PlantCard>
+            />
+            <BookmarkButton
+              plantId={plant.id}
+              onToggleOwned={onToggleOwned}
+              ownedPlantsIds={ownedPlantsIds}
+            ></BookmarkButton>
```

Comment:
Shoudn't the BookmarkButton be part of the PlantCard? In that case the button would be also rendered on the bookmark-page. At the moment you don't have the button there.

If an element does not have content you should prefer self closing tags.